src/ansys/pyensight/core/utils/export.py

```diff
@@ -464,3 +462,100 @@ def _animation_remote(
                 mp4_data = fp.read()
 
         return mp4_data
+
+    GEOM_EXPORT_GLTF = "gltf2"
+    GEOM_EXPORT_AVZ = "avz"
+    GEOM_EXPORT_PLY = "ply"
+    GEOM_EXPORT_STL = "stl"
+
+    extension_map = {
+        GEOM_EXPORT_GLTF: ".glb",
+        GEOM_EXPORT_AVZ: ".avz",
+        GEOM_EXPORT_PLY: ".ply",
+        GEOM_EXPORT_STL: ".stl",
+    }
+
+    def _geometry_remote(
+        self, format: str, begin_timestep: int = 0, end_timestep: int = 0, delta_timestep: int = 1
+    ) -> bytes:
+        """EnSight-side implementation.
+
+        Parameters
+        ----------
+        format : str
+            The format to export
+        begin_timestep: int
```

Comment:
General comment on the timesteps limits.  I kind of prefer the keywords:

starting_timestep
number_timesteps

(or 'frames').  The reason for this factorization is that for 95% of the case, I don't want users to have to figure out how many timesteps there are.  IMHO, the two common cases are:

1) all of the timesteps
2) the current timestep

With the current API, you need to look both of these up.  If you were to default the number_timesteps to 1 and default the starting_timestep to the current one, that handles case (2) w/o any keywords being set.  case (1) would only require that starting_timestep be set to 0 and the count be set to  ensight.objs.core.TIMESTEP_LIMITS[1].   An "nice" feature might be if number_timesteps is set to -1 or None explicitly, then we compute it to be from the start point to the last timestep.  Just trying to make sure the user can get away with setting as few of the keywords as possible. 

Another spin on this is that the default for the number of timesteps is None which is mapped to all of them, from the current timestep and the user can pretty easily set that to '1' for the curent timest, etc.

Reply:
So, I have changed the parameters like this:

        filename: str
            The location where to export the geometry
        format : str
            The format to export
        starting_timestep: int
            The first timestep to export. If None, defaults to the current timestep
        frames: int
            Number of timesteps to save. If None, defaults from the current timestep to the last
        delta_timestep: int
            The delta timestep to use when exporting

Reply:
self,
        filename: str,
        format: str = GEOM_EXPORT_GLTF,
        starting_timestep: Optional[int] = None,
        frames: Optional[int] = 1,
        delta_timestep: Optional[int] = None

Reply:
if not starting_timestep:
            starting_timestep = int(self._ensight.objs.core.TIMESTEP)
        if not frames or frames == -1:
            # Timesteps are 0-indexed so frames need to be increased of 1
            frames = int(self._ensight.objs.core.TIMESTEP_LIMITS[1]) + 1
        if not delta_timestep:
            delta_timestep = 1